docker_files/home/lisk/run.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+confd -backend env -onetime
+node app.js
```

Comment:
Why we are not using `lisk.sh` script here, if possible we should use it for consistency across distribution platforms.

Reply:
We are not using `lisk.sh` here because 
- it is meant to manage an all-in-one installation which this is not and
- it is meant to be run interactively: it starts processes and then returns which is not what the PID 1 of a container should do